src/main/scala/is/hail/io/vcf/LoadVCF.scala

```diff
@@ -119,9 +199,17 @@ final class VCFLine(line: String) {
   def nextField(): Unit = {
     if (pos == line.length)
       fatal("unexpected end of line")
+    assert(line(pos) == '\t')
     pos += 1 // tab
   }
 
+  def nextFormatField(): Unit = {
+    if (pos == line.length)
+      fatal("unexpected end of line")
+    assert(line(pos) == ':')
```

Comment:
won't generate a nice error on malformatted VCF

Reply:
I think this is OK.  `nextFormatField` should only be called if `endField` is not true: that means we parsed a field and we're not done with the format field, so the only thing the current character can be is a ':'.

Reply:
ah, alright. Fair enough.